package.json

```diff
@@ -6,15 +6,16 @@
   "repository": "github:Lambda-School-Labs/labs-api-starter",
   "scripts": {
     "start": "node server.js",
-    "migrate": "knex run:migrate",
-    "rollback": "knex run:rollback",
+    "migrate": "knex migrate:latest",
```

Comment:
Nice touch here in the scripts, had to google knex cmd line commands to get these to work on local dev, this will be really useful.